src/nodedef.h

```diff
@@ -790,6 +792,12 @@ class NodeDefManager {
 	 */
 	void fixSelectionBoxIntUnion();
 
+	/*!
+	 * Calculate ContentFeatures bigSelectonBox value.
+	 * m_selection_box_union.
```

Comment:
Please touch this comment up a bit. I think just `/// Calculate ContentFeatures bigSelectionBox value.` would suffice.